src/components/LikedPage/LikedVideoCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useHistory } from "../../context/historyContext";
+import { useLikedVideo } from "../../context/likedVideoContext";
+import { useWatchLater } from "../../context/watchLaterContext";
+
+const LikedCard = ({ item, watchLaterVideos, LikedVideos }) => {
+  const [dropdown, setDropdown] = useState(false);
+  const trimHeading = (word, n) => {
```

Comment:
you can move this function out of component as it doesn't need to get initialized on every render.